### ISSUE TITLE
invalid install.xml file.

### ISSUE THREAD

**Issue by danmarsden** [adlnet/cmi5-player-moodle-plugin#30]
The install.xml is no longer valid and was either created for an old version of Moodle or has been handwritten.

This needs to be fixed using the Moodle xmldb editor (under admin > development > XMLDB.)

Please hit the "load" link for your plugin install.xml file in the moodle xmldb editor, and then make Moodle think there is an update to the file (easiest way to do this is to change the description of a table - add a full stop or something similar.) Then get XMLDB to "Save" the updated file which will remove all the old invalid xml from your file.

**Comment by ADLMeganBohland** [comment]
@danmarsden , Hey Dan, I did as you said and I think it worked? Is it valid now?

**Comment by danmarsden** [comment]
yeah that looks correct to me - sorry probably should have said that this one wasn't a blocker for approval - I just spotted some spacing that didn't come from xmldb...  :-)

**Comment by ADLMeganBohland** [comment]
Oh that's ok, thanks!
